extensions/sql-bindings/src/common/constants.ts

```diff
@@ -80,3 +80,4 @@ export const enterPasswordManually = localize('enterPasswordManually', 'Enter pa
 export const userPasswordLater = localize('userPasswordLater', 'In order to user the SQL connection string later you will need to manually enter the password in your local.settings.json file.');
 export const openFile = localize('openFile', "Open File");
 export const closeButton = localize('closeButton', "Close");
+export const addSqlBinding = localize('addSqlBinding', "Adding SQL Binding");
```

Comment:
Let's add the function name here too `Adding SQL binding to function 'My Function'...`